Libraries/pbxbuild/Sources/Tool/CopyResolver.cpp

```diff
@@ -77,13 +78,20 @@ ResolveInternal(
     Tool::OptionsResult options = Tool::OptionsResult::Create(toolEnvironment, toolContext->workingDirectory(), nullptr);
     Tool::Tokens::ToolExpansions tokens = Tool::Tokens::ExpandTool(toolEnvironment, options, std::string(), args);
 
+    std::vector<std::string> toolInputPaths = toolEnvironment.inputs(toolContext->workingDirectory());
+
     // TODO(grp): This should be generic for all tools.
     std::vector<Tool::Invocation::DependencyInfo> dependencyInfo;
     if (tool->deeplyStatInputDirectories()) {
         for (std::string const &inputPath : inputPaths) {
             /* Create a dependency info file to track the input directory contents. */
-            auto info = Tool::Invocation::DependencyInfo(dependency::DependencyInfoFormat::Directory, inputPath);
-            dependencyInfo.push_back(info);
+            libutil::DefaultFilesystem filesystem = libutil::DefaultFilesystem();
```

Comment:
I think that makes sense, but if the dependency tool ignores files, then the dependency is going to be broken, no? Maybe instead of ignoring files, the tool should actually handle them properly and output info for a single file?